pkg/capture/capture_test.go

```diff
@@ -57,7 +57,7 @@ func (t testMockSrcs) Wait() error {
 func TestConcurrentMethodAccess(t *testing.T) {
 	for _, i := range []int{1, 2, 3, 10} {
 		t.Run(fmt.Sprintf("%d ifaces", i), func(t *testing.T) {
-			testConcurrentMethodAccess(t, i, 1000)
+			testConcurrentMethodAccess(t, i, 100)
```

Comment:
Had to reduce this because the local buffer logic introduces additional overhead that actually slows down the code if you try it as excessively as this test does (everything, everywhere, all at once)...